### PR TITLE
Body Mass Index

### PR DESCRIPTION
**Pull request by alfonsserra**
Use Body Mass Index for the unit test example instead of the CV calculation.

### CODE REVIEW

**Comment by alfonsserra** [comment]
Good Catch @jazzinjars. The test has been upgrade to JUnit 5.

File: src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        Double bmiCalculated = BodyMassIndexCalculator.getBMI(scenario.weight, scenario.height);
+        if (scenario.expectedResult == null)
+            TestUtil.checkObjectIsNull("BMI", bmiCalculated);
+        else
+            TestUtil.checkField("BMI", scenario.expectedResult, bmiCalculated);
+    }
+
+    //This is an example of a Single Test in the same class with Parameterized Tests
+    @TmsLink("TC1234_Formulas")
+    @Feature("Goal: This test case is intended to verify the correct calculation of the different Statistical Calculations")
```
Comment (olgapuig) [inline src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java]:
```suggestion
    @Feature("Goal: This test case is intended to verify the correct calculation of the Body Mass Index")
```

File: src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
+package com.systelab.seed.patient;
```
Comment (olgapuig) [inline src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java]: Change the package to be the same as the source code.
```suggestion
package  com.systelab.seed.patient.control;
```

File: src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        public Double expectedResult;
+
+        public Scenario(Double weight, Double height, Double expectedResult) {
+            this.weight = weight;
+            this.height = height;
+            this.expectedResult = expectedResult;
+        }
+    }
+
+    static Scenario[] scenarios() {
```
Comment (olgapuig) [inline src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java]: Use Arrays.asList
```suggestion
    static List<Scenario> scenarios() {
```

File: src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        };
+        // @formatter:on
+    }
+
+    @Step("Action: With a weight of {0} kg and a height of {1} meters")
+    public void printScenario(Double weight, Double height) {
+    }
+
+    @TmsLink("TC1234_Formulas")
+    @Feature("Goal: This test case is intended to verify the correct calculation of the different Statistical Calculations")
```
Comment (olgapuig) [inline src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java]:
```suggestion
    @Feature("Goal: This test case is intended to verify the correct calculation of the Body Mass Index")
```

File: src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                new Scenario(45.0, 1.0, 45.0),
+                new Scenario(65.0, 1.40, 33.2),
+                new Scenario(75.0, 1.8, 23.1),
+                new Scenario(95.0, 1.8, 29.3),
+                new Scenario(78.5, 1.805, 24.1)
+        };
+        // @formatter:on
+    }
+
+    @Step("Action: With a weight of {0} kg and a height of {1} meters")
```
Comment (olgapuig) [inline src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java]:
```suggestion
    @Step("Action: Calculate the BMI with a weight of {0} kg and a height of {1} meters")
```

File: src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+        public Scenario(Double weight, Double height, Double expectedResult) {
+            this.weight = weight;
+            this.height = height;
+            this.expectedResult = expectedResult;
+        }
+    }
+
+    static Scenario[] scenarios() {
+        // @formatter:off
+        return new Scenario[]{
```
Comment (olgapuig) [inline src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java]:
```suggestion
        return Arrays.asList(
```

File: src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+                new Scenario(1.0, -1.0, null),
+                new Scenario(1.0, 0.0, null),
+                new Scenario(5.0, 0.1, null),
+                new Scenario(30.0, 0.5, 120.0),
+                new Scenario(45.0, 1.0, 45.0),
+                new Scenario(65.0, 1.40, 33.2),
+                new Scenario(75.0, 1.8, 23.1),
+                new Scenario(95.0, 1.8, 29.3),
+                new Scenario(78.5, 1.805, 24.1)
+        };
```
Comment (olgapuig) [inline src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java]:
```suggestion
         );
```

**Pull request merged**